code/__HELPERS/roundend.dm

```diff
@@ -443,6 +429,50 @@ GLOBAL_LIST_INIT(achievements_unlocked, list())
 
 	return parts.Join()
 
+/// This is gives the player a little quip about the fate of ther character after the events on the station.
+/datum/controller/subsystem/ticker/proc/get_survial_text(mob/survivor)
+	var/list/survival_text = list()
+	var/green_border = "<div class='panel greenborder'>"
+	var/red_border = "<div class='panel redborder'>"
+
+	if(survivor.stat == DEAD || isbrain(survivor))
+		survival_text += red_border
+		survival_text += span_redtext("You did not survive the events on [station_name()]...")
+		return survival_text
+
+	if(survivor.onCentCom()||!EMERGENCY_ESCAPED_OR_ENDGAMED) //Never going back to that place again.
+		survival_text += green_border
+		survival_text += span_greentext("You managed to survive the events on [station_name()] as [survivor.real_name].")
+		return survival_text
+
+	if(survivor.onSyndieBase() && SSshuttle.emergency.is_hijacked())
+		if(survivor.mind.get_desired_hijack_destination() == SYNDICATE_BASE)
+			survival_text += green_border
+			survival_text += "<span class='greentext'>You managed to hijack the emergency shuttle of [station_name()] and bring aboard the syndicate base!</span>"
+		else
+			survival_text += red_border
+			survival_text += "<span class='redtext'>You managed to survive the events of [station_name()] as [survivor.real_name], but were captured by the syndicate!</span>"
+		return survival_text
+
+	if(SSshuttle.emergency.is_hijacked() && survivor.on_escaped_shuttle())
```

Comment:
If you're planning to continue expanding this with more options you might want to delegate it to a proc, perhaps even with some form of hijack datum to provide the correct text.
In either case it also would probably make more sense to do:
```suggestion
	if(SSshuttle.emergency.is_hijacked())
		if(survivor.onSyndieBase())
			if(survivor.mind.get_desired_hijack_destination() == SYNDICATE_BASE)
				survival_text += green_border
				survival_text += "<span class='greentext'>You managed to hijack the emergency shuttle of [station_name()] and bring aboard the syndicate base!</span>"
			else
				survival_text += red_border
				survival_text += "<span class='redtext'>You managed to survive the events of [station_name()] as [survivor.real_name], but were captured by the syndicate!</span>"
			return survival_text

		if(survivor.on_escaped_shuttle())
			...
```